.devcontainer/devcontainer.json

```
@@ -1,4 +1,5 @@
 {
+  "image": "mcr.microsoft.com/devcontainers/base:jammy",
```

Would this be better?

```suggestion
  "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
```